src/components/PopupWithForm.js

```diff
@@ -32,6 +33,14 @@ class PopupWithForm extends Popup {
     });
     super.setEventListeners();
   }
+
+  renderLoading(isLoading) {
```

Comment:
You've implemented the logic for rendering loading via the class methods!

Reply:
Thank you!